### PR TITLE
feat: Allow commit message ONLY with --messageonly

### PR DESCRIPTION
**Pull request by vespasianvs**
For my project we needed the commit messages only (with no URLs, author names etc). I added the functionality for my project and thought I'd submit it here too to see if it is useful for others. Follows the same pattern as `--plaintext` but displays the commit messages only.

Also changed the grouping so that it doesn't display the single colon if the commitGroup value is undefined/empty.

### CODE REVIEW

**Comment by rvagg** [comment]
lgtm, there's a test failure with a simple fix as I've suggested, also please remove the package-lock.json from the commit

**Comment by vespasianvs** [comment]
Apologies - forgot to check back on this - will take a look

**Comment by vespasianvs** [comment]
Think that should do it? Finding it hard to run the tests locally as I'm on Windows 11 - even using WSL (Ubuntu) the tests aren't passing properly all of the time. Use the output from the workflow to fix the tests - I hope.

**Comment by rvagg** [comment]
Yeah, sorry, windows support is lacking, there was an attempt to do it at one point but that was never completed. You're welcome to give that a try but iirc it was kind of gnarly. Some of that history can be traced through here: https://github.com/nodejs/changelog-maker/pull/66

File: commit-to-output.js [hunk 1]
```diff
@@ -90,6 +91,12 @@ function toStringMarkdown (data) {
         : s)
 }
 
+function toStringMessageOnly (data) {
+  let s = ''
+  s += data.summary
+  return `  * ${s.trim()}`
```
Comment (rvagg) [inline commit-to-output.js:97]:
```suggestion
  return `  * ${data.summary.trim()}`
```

**Comment by rvagg** [comment]
lgtm except for that one little change to remove unnecessary lines

**Comment by vespasianvs** [comment]
Lines removed. I know exactly what I did there - took the toStringPlaintext and removed the lines I didn't want - then didn't spot the shortcut :)

If I can find time, I'll have a look at the Windows tests - I already had a branch open to look at it but ran out of time.

**Pull request merged**